src/rendering/RenderUtil.cc

```diff
@@ -734,6 +784,25 @@ void RenderUtil::Update()
       }
     }
   }
+
+  // update thermal camera
+  for (const auto &thermal : this->dataPtr->thermalCameraData)
+  {
+    Entity id = thermal.first;
+    rendering::ThermalCameraPtr camera =
+        std::dynamic_pointer_cast<rendering::ThermalCamera>(
+        this->dataPtr->sceneManager.NodeById(id));
+    if (camera)
+    {
+      double resolution = std::get<0>(thermal.second);
+      if (resolution > 0.0)
+        camera->SetLinearResolution(resolution);
+      double minTemp = std::get<1>(thermal.second).min.Kelvin();
+      double maxTemp = std::get<1>(thermal.second).max.Kelvin();
+      camera->SetMinTemperature(minTemp);
+      camera->SetMaxTemperature(maxTemp);
```

Comment:
```suggestion
      camera->SetMinTemperature(std::min(minTemp, maxTemp));
      camera->SetMaxTemperature(std::max(minTemp, maxTemp));
```

In the case where a user may specify `min > max`, we should check for this before setting the camera temperature parameters. Using `std::min` or `std::max` requires [`#include <algorithm>`](https://en.cppreference.com/w/cpp/header/algorithm).

Reply:
I added a check to make sure max >= min. ee13143